lib/dal/src/schema/variant/authoring.rs

```diff
@@ -302,49 +300,52 @@ impl VariantAuthoringClient {
         }
     }
 
-    #[allow(clippy::too_many_arguments)]
-    #[instrument(name = "variant.authoring.update_variant", level = "info", skip_all)]
+    #[instrument(
+        name = "variant.authoring.regenerate_variant",
```

Comment:
This was subtle, but would have been a pain later.
cc @stack72 @johnrwatson @sprutton1

Reply:
Nice catch